composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0",
+        "php": "^7.2 || ^8.0",
```

Comment:
I would prefer to go with 8.x. You can see 7.x active support already ended. 

![image](https://user-images.githubusercontent.com/120454/151136834-41d7fa35-f0e5-4b43-b3e0-2720b1627236.png)

Reply:
> I would prefer to go with 8.x. You can see 7.x active support already ended.

The telemetry from my application (approx 3000 installations) shows the overwhelming majority of sites are still using PHP 7.4

<img width="475" alt="Screenshot 2022-01-26 at 10 29 59" src="https://user-images.githubusercontent.com/2306088/151147484-2c36656e-4d00-4061-8908-d57105461427.png">

Is there a technical reason to drop support for PHP 7.4?

Aura runs on a very wide range of PHP versions, which makes it great to use in applications like mine, which will be deployed on a wide range of servers.

Could it not be dropped in a later release (e.g. 4.1.)

Reply:
We will reconsider this. :) .

Reply:
@harikt　Here's what @pmjones had to say.
https://github.com/auraphp/Aura.Cli_Kernel/pull/12#issuecomment-873248524
 It's easy to keep PHP 7.2 to 8.1 compatible with a single version of PHP Unit.

Reply:
Cool